proto/ate.proto

```diff
@@ -643,6 +643,11 @@ message Ipv4Header {
   uint32 dscp = 4;
   uint32 ecn = 5;
   uint32 checksum = 7;
+  UIntRange protocol = 8;
+  uint32 identification = 9;
+  bool reserved = 10;
```

Comment:
I'm guessing there is no need to ever set the evil bit to 1, so I'll drop this when incorporating this PR. Please let me know if it is really needed.